packages/editor/src/components/post-taxonomies/hierarchical-term-selector.js

```diff
@@ -428,8 +428,7 @@ export function HierarchicalTermSelector( { slug } ) {
 			{ ! loading && hasCreateAction && (
 				<FlexItem>
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
<table>
<tr>
<td>Before</td>
<td>After</td>
</tr>
  <tr>
    <td>
<img width="279" alt="term selector before" src="https://github.com/user-attachments/assets/f98c89ed-fdd0-449d-ae8f-d4eada176ea1">


</td>
    <td>
<img width="275" alt="term selector after" src="https://github.com/user-attachments/assets/83e223f6-6f1c-4562-aadb-b8c0ff31f11d">


</td>
  </tr>
</table>

Note:-  button variant set to link.

Reply:
Yeah, that makes sense. We'll make sure to exclude these when we start throwing deprecation warnings in console.